docs/how-to-guides/add-new-dashboards-new-alerts.md

```diff
@@ -0,0 +1,93 @@
+# Add Custom Monitoring Configs
```

Comment:
```suggestion
# Add custom monitoring configs
```
and others

Reply:
Since this is a title, shouldn't it be capitalised?

Reply:
We follow the `Sentence case`. https://github.com/kinvolk/style-guide/blob/master/style-guide.md#title-styling